srcpkgs/radicale/INSTALL.msg

```diff
@@ -0,0 +1,13 @@
+We also provide radicale 2.x.x
```

Comment:
i'd rather not nag radicale1 users

Reply:
I am curious: why?
Radicale 1.x depends on python2 and last release and commit was in nov. 2017.
I don't see the point in offering an outdated tool without mentioning that its superseded.